src/streamlink/plugins/cnews.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HDSStream, HLSStream, HTTPStream
+
+
+class CNEWS(Plugin):
+    _url_re = re.compile(r'(https|http)://www.cnews.fr(/\w)+')
+    _embed_video_url_re = re.compile(r'class="dm-video-embed_video" src="(?P<dm_url>.*)"')
+    _embed_live_url_re = re.compile(r'class="wrapper-live-player main-live-player"><iframe src="(?P<dm_url>.*)"')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return CNEWS._url_re.match(url)
```

Comment:
not important, but you could just use `cls` instead of `CNEWS` here

Reply:
More important when subclassing, so a good habit to get in to :)